### PR TITLE
[VEN-2519]: Configure XVS Bridge Ethereum <-> opBNB

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-2519

### CODE REVIEW

**Comment by GitGuru7** [comment]
Sepolia: https://sepolia.etherscan.io/address/0x94fa6078b6b8a26F0B6EDFFBE6501B22A10470fB

opBNB testnet: https://testnet.opbnbscan.com/tx/0x5664dc41dd74190a70cf7b846ee45b621a9552264877d19e73d7726486296584

Sepolia -> opBNB testnet: https://testnet.layerzeroscan.com/tx/0xd9a73ac04a69077f5eacc59551c84b9ccc062f69dec050db5669d6463dddb1af

opBNB testnet -> Sepolia: https://testnet.layerzeroscan.com/tx/0xd0e6e32e1d006127b5e6c3d9dbb06f1b6f24eccbbcf810834e492db8ac715c57

File: vips/vip-292/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  return makeProposal(
+    [
+      {
+        target: FAST_TRACK_TIMELOCK,
+        signature: "",
+        params: [],
+        value: "1",
+      },
```
Comment (chechu) [inline vips/vip-292/bscmainnet.ts:23]: I think we should replace this command by the commands to set the new receive limits from Ethereum and opBNB, setting the 2% margin respect to the sending limits

Comment (GitGuru7) [reply vips/vip-292/bscmainnet.ts]: [ad78301](https://github.com/VenusProtocol/vips/pull/255/commits/ad78301dc961010f4e14e0d9e77d45dd7c331fa9)

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/292?chainId=56

**Pull request merged**